tempesta_fw/http_sess.c

```diff
@@ -413,7 +413,7 @@ tfw_http_sticky_verify(TfwHttpReq *req, TfwStr *value, StickyVal *sv)
 	if (__sticky_calc(req, sv))
 		return TFW_BLOCK;
 	for (i = 0, hi = 1; (c) < end; ++(c)) {
-		for ( ; p < (unsigned char *)c->ptr + c->len; ++p) {
+		for ( ; p < (unsigned char *)c->data + c->len; ++p) {
```

Comment:
If `c->data` is used just as a pointer, is there the need to cast it from `(char *)` to `(unsigned char *)`?